web/satellite/src/components/accessGrants/AccessGrants.vue

```diff
@@ -326,6 +340,41 @@ export default class AccessGrants extends Vue {
     public get selectedAccessGrantsAmount(): number {
         return this.$store.state.accessGrantsModule.selectedAccessGrantsIds.length;
     }
+
+    /**
+     * Access grant button click.
+     */
+    public accessGrantClick(): void {
+        this.modalPropType = 'access'
+        this.toggleAccessModal()
+    }
+
+    /**
+     * S3 Access button click..
+     */
+    public s3Click(): void {
+        this.modalPropType = 's3'
+        this.toggleAccessModal()
+    }
+
+    /**
+     * CLI Access button click.
+     */
+    public cliClick(): void {
+        this.modalPropType = 'api'
+        this.toggleAccessModal()
+    }
+
+    /**
+     * toggles Create Access Modal visibility.
+     */
+    public toggleAccessModal(): void {
+        if (this.showModal === false) {
```

Comment:
this.showModal = !this.showModal

Reply:
resolved
`
    /**
     * toggles Create Access Modal visibility.
     */
    public toggleAccessModal(): void {
        if (this.showAccessModal = !this.showAccessModal) {
`

Reply:
resolved